### PR TITLE
Change error message for sample

### LINKED ISSUE #345: Sample Command gives incorrect message for neg values and values > 100
image

Should be just statement showing incorrect range of values used

### PR DESCRIPTION
**Pull request by kennycjy**
Basically changed the phrasing of the message usage.

fixes #345

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SampleCommand.java [hunk 1]
```diff
@@ -20,8 +20,8 @@ public class SampleCommand extends Command {
     public static final String COMMAND_WORD = "sample";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Populate the EduMate with "
-            + "sample data of the given size.\n"
-            + "Parameters: SIZE (non-negative integer)\n"
+            + "sample data of the given size between 1 and 100.\n"
+            + "Parameters: SIZE (non-negative integer less than 100)\n"
```
Comment (russelltankaimin) [inline src/main/java/seedu/address/logic/commands/SampleCommand.java:24]: these 2 sentences contradict each other,

**Review by russelltankaimin — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/SampleCommand.java (hunk 1 above)
Comment (kennycjy) [reply src/main/java/seedu/address/logic/commands/SampleCommand.java]: Just  changed to 101.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -78,8 +80,13 @@ public static ContactIndex parseContactIndex(String contactIndex) throws ParseEx
      */
     public static int parseInt(String intString) throws ParseException {
         String trimmedInt = intString.trim();
-        if (trimmedInt.isEmpty() || !StringUtil.isNonZeroUnsignedInteger(trimmedInt)) {
-            throw new ParseException(MESSAGE_INVALID_INDEX);
+        if (trimmedInt.isEmpty()) {
+            throw new ParseException(MESSAGE_INVALID_INT);
+        }
+        try {
+            Integer.parseInt(trimmedInt);
```
Comment (russelltankaimin) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:87]: `Integer integer = Integer.parseInt(trimmedInt);`
would have done the same, just dont need to parse again.

File: src/main/java/seedu/address/logic/parser/SampleCommandParser.java [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-
 import seedu.address.logic.commands.SampleCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
  * Parses input arguments and creates a new SampleCommand object
  */
 public class SampleCommandParser implements Parser<SampleCommand> {
+    public static final String INVALID_SIZE = "Invalid size. Please select a number between 1 and 100 inclusive.";
```
Comment (russelltankaimin) [inline src/main/java/seedu/address/logic/parser/SampleCommandParser.java:10]: Rename to MESSAGE_INVALID_SIZE for standardisation

File: src/main/java/seedu/address/logic/parser/SampleCommandParser.java [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
     @Override
     public SampleCommand parse(String args) throws ParseException {
         try {
             int size = ParserUtil.parseInt(args);
-            if (size < 0 || size > 100) {
-                throw new ParseException("Invalid size. Please select a number between 1 and 100 inclusive.");
+            if (size < 1 || size > 100) {
+                throw new ParseException(String.format(INVALID_SIZE));
+                // problem here is that it doesn't make use of the above line.
             }
```
Comment (russelltankaimin) [inline src/main/java/seedu/address/logic/parser/SampleCommandParser.java:19]: is this line still valid? otherwise please remove it.

**Review by russelltankaimin — CHANGES_REQUESTED** [review]
Please make the small changes. Thanks

**Review by russelltankaimin — APPROVED** [review]
LGTM

**Pull request merged**